### DISCUSSION TITLE
Devices not updating

### DISCUSSION THREAD

**Discussion by jloffredo** [mKeRix/room-assistant#1080] [General]
Hi, I've installed room-assistant in a raspberry Pi. Configured MAC for a OnePlus 7T phone and a Samsung Gear S3 Frontier watch. Both get detected and home assistant MQTT devices are created if I turn off and on the bluetooth on both devices, and get updated. But if I leave bluetooth on and use as normal, Room-Assistant doesn't detect or update them.

I'm using the base configuration. 
Any idea as to what I could try to fix this? Thanks!

**Comment by drewancil** [comment]
I am having the exact same issue.

**Comment by phoniclynx** [comment]
Yeah same here.. it's frustrating the life out of me... this could be such a great project, but it doesn't seem to want to work as seamlessly as it should :(
